scripts/tablet-mon.py

```diff
@@ -0,0 +1,637 @@
+#!/bin/env python
+#
+# Usage:
+#
+#   ./tablet-mon.py <node-ip>
+#
+
+import sys
+import math
+import threading
+import time
+import logging
+import pygame
+
+from cassandra.cluster import Cluster
+
+# Layout settings
+tablet_size = 60
+node_frame_size = 18
+node_frame_thickness = 3
+frame_size = 30
+shard_spacing = 0
+node_frame_mid = node_frame_size // 2
+
+tablet_w = tablet_size
+tablet_h = tablet_w // 2
+tablet_radius = tablet_size // 7
+tablet_frame_size = max(2, min(6, tablet_size // 14))
+
+# Animation settings
+streaming_trace_duration_ms = 300
+streaming_trace_decay_ms = 2000
+streaming_done_trace_duration_ms = 40
+streaming_done_trace_decay_ms = 1000
+insert_time_ms = 500
+fall_acceleration = 3000
+fall_delay_ms = 0
+
+# Displayed objects
+nodes = []
+nodes_by_id = {}
+tracers = list()
+trace_lines = set()
+trace_lines_pool = list()
+
+cql_update_timeout_ms = 800
+cql_update_period = 0.2
+
+
+class Node(object):
```

Comment:
in Python 3, objects are so-called new-style object by default, no need to inherit from `object` anymore.

Reply:
Yes. I changed to shebang to python3 and dropped the inheritance.